docs/product/requirements/2025/sso-oidc.md

```diff
@@ -37,91 +37,14 @@ SSO may also help us grow Mathesar adoption, because:
 
 ### Is it feasible to solve?
 
-Yes! We use Django for our users, and there are several libraries we can just drop into our codebase.
+Yes! We use Django for our users.
 
 * [juanifioren/django-oidc-provider](https://github.com/juanifioren/django-oidc-provider)  
 * [jazzband/django-oauth-toolkit](https://github.com/jazzband/django-oauth-toolkit)  
 * [mozilla/mozilla-django-oidc](https://github.com/mozilla/mozilla-django-oidc)  
 
 Further feasibility research is unnecessary, it should be quick to implement. I briefly considered just implementing it instead of writing this.
```

Comment:
What is the intention behind removing "and there are several libraries we can just drop into our codebase."? Is it to be less prescriptive about a specific solution? I find "Yes! We use Django for our users." followed by a list of libraries confusing, as there's no context for what the list of libraries means.

Reply:
Ah I see that the line is also in the project proposal doc so I must've had some harebrained thought about differentiating between the two.